make.js

```diff
@@ -1021,11 +1034,63 @@ CLI.gentask = function() {
             console.log(`Running \"npm install\" command in ${taskPath}`);
             run(`npm install`);
             cd(__dirname);
+
+            // copy package.json, package-lock.json and npm-shrinkwrap.json files from generated tasks to the Tasks\taskname\_buildConfig\nodeversion folder.
+            const fileNames = ['package.json', 'package-lock.json', 'npm-shrinkwrap.json'];
```

Comment:
same question as here: https://github.com/microsoft/azure-pipelines-tasks/pull/18933#discussion_r1322554408

Reply:
Files are not getting copied after adding syncGeneratedFilesWrapper here, probably something different between logics here. I'll keep the logics here for consistency and get back to this.